Kickstarter-iOS/Views/Controllers/SettingsViewController.swift

```diff
@@ -172,9 +172,9 @@ internal final class SettingsViewController: UIViewController {
     _ = self.emailNotificationButtons
       ||> settingsNotificationIconButtonStyle
       ||> UIButton.lens.image(forState: .normal)
-        .~ UIImage(named: "email-icon", in: .framework, compatibleWith: nil)
+        .~ UIImage(named: "icon--mail", in: .framework, compatibleWith: nil)
```

Comment:
Hey @cdolm92 was just wondering why these are double hyphen now? I think our convention is to use the single hyphens.